dygraph/ppdet/engine/trainer.py

```diff
@@ -95,21 +103,20 @@ def _init_callbacks(self):
             self._compose_callback = None
 
     def _init_metrics(self):
-        if self.mode == 'eval':
-            if self.cfg.metric == 'COCO':
-                self._metrics = [COCOMetric(anno_file=self.dataset.get_anno())]
-            elif self.cfg.metric == 'VOC':
-                self._metrics = [
-                    VOCMetric(
-                        anno_file=self.dataset.get_anno(),
-                        class_num=self.cfg.num_classes,
-                        map_type=self.cfg.map_type)
-                ]
-            else:
-                logger.warn("Metric not support for metric type {}".format(
-                    self.cfg.metric))
-                self._metrics = []
+        if self.cfg.metric == 'COCO':
+            mask_resolution = self.model.mask_post_process.mask_resolution if getattr(
```

Comment:
mask_resolution is not used in metric now, mask post_process is moved to the model

Reply:
Done, thanks!